.github/workflows/alpha-promotion.yaml

```diff
@@ -0,0 +1,49 @@
+name: Alpha Nightly Promotion
+on:
+  pull_request
+#  schedule:
+#    - cron: '30 2 * * *' # run at 2:30 AM UTC
+env:
+  chartVersion: 0.15.0
+  uiVersion: 0.0.6
+
+jobs:
+  alpha_promotion:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: 'Get Previous tag'
+        id: apiprevioustag
+        uses: "broadinstitute/github-action-get-previous-tag@master"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      - name: 'Checkout terra-helmfile repo'
+        uses: actions/checkout@v2
+        with:
+          repository: 'broadinstitute/terra-helmfile'
+          token: ${{ secrets.HELM_REPO_TOKEN }}
+          path: terra-helmfile
+      - name: "Find and replace Datarepo version in versions.yaml"
+        uses: docker://mikefarah/yq:latest
+        with:
+          args: yq w -i terra-helmfile/versions.yaml releases.datarepo.appVersion ${{ steps.apiprevioustag.outputs.tag }}
+      - name: "Find and replace Datarepo chartVersion version in versions.yaml"
+        uses: docker://mikefarah/yq:latest
+        with:
+          args: yq w -i terra-helmfile/versions.yaml releases.datarepo.chartVersion ${{ env.chartVersion }}
+      - name: "Read terra-helmfile daterepo fields in versions.yaml"
+        uses: docker://mikefarah/yq:latest
```

Comment:
Sorry to be lame, but is there a more descriptive docker path we could use?

Reply:
i don't follow? mikefarah is the official releaser of yq

Reply:
Gotcha! I was wrong.